requirements.txt

```
@@ -1,5 +1,6 @@
 ansible_runner<2.0; python_version < '3.6'
 ansible_runner; python_version >= '3.6'
+black
```

What about we restrict this to newer pythons
```suggestion
black; python_version >= '3.6'
```

---

Nice idea.